packages/terra-date-picker/src/DatePicker.scss

```diff
@@ -0,0 +1,54 @@
+@import '~terra-mixins';
+@import '../node_modules/react-datepicker/dist/react-datepicker.min';
+
+.terra-DatePicker {
+  display: inline-block;
+  min-width: 14.643em; 
+}
+
+.terra-DatePicker-input {
+  border: 0.071em solid $terra-grey-40;
+  box-shadow: inset 0 0.143em 0.143em $terra-grey-20;
+  font-size: 1em;
+  height: 1.143em;
+  line-height: 1.143em;
+  margin: 0.357em 0;
+  padding: 0.429em 0.714em;
+  vertical-align: middle;
+
+  @if $terra-bidi {
+    [dir='ltr'] & {
+      border-radius: 0.286em 0 0 0.286em;
```

Comment:
Updated to use the mixins: https://github.com/cerner/terra-core/commit/c599197256ce3c511ea206ca5e87c4015f4f6fee